src/batch-client/index.ts

```diff
@@ -1352,6 +1353,29 @@ export class ZimbraBatchClient {
 		// If account name is present that means we will not be able to batch requests
 		this[options.singleRequest ? 'dataLoader' : 'batchDataLoader'].load(options);
 
+	public listDocumentRevisions = ({ id, version, count }: any) =>
+		this.jsonRequest({
+			name: 'ListDocumentRevisions',
+			namespace: Namespace.Mail,
+			body: {
+				doc: {
+					id,
+					ver: version,
+					count
+				}
+			}
+		}).then(response => {
+			const data = normalize(ListDocumentRevisions)(response);
+			const versionIds = data.documents.map((doc: any) => doc.version);
+			const maxVersion = Math.max(...versionIds);
```

Comment:
I think version ids are in string format, how does Math.max works with string data? does it convert to int implicitly?
for clarity we should convert these version ids to int ourself

Reply:
Version ids are in Int format.